### PR TITLE
Vt allow undefined env field

### PR DESCRIPTION
**Pull request by vtj2105**
This PR allows manifests without both environment fields and public/private var fields to pass validation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "codius-manifest",
-  "version": "2.0.2",
+  "version": "2.0.3",
```
Comment (traviscrist) [inline package.json]: Can you revert this?  We should be doing the version changes are part of a tagging process.

File: src/common/check-usage.js [hunk 2]
```diff
@@ -8,7 +8,9 @@ const checkUsage = function (manifest, varName) {
   for (let i = 0; i < containers.length; i++) {
     const container = containers[i]
     const envVars = container['environment']
-
+    if (!envVars) {
```
Comment (justmoon) [inline src/common/check-usage.js]: I would slightly prefer this:

```js
const envVars = container['environment'] || {}
```

If somebody changes this section in the future, they could easily miss the `continue`. It's good to minimize the number of different paths there are through your code.

File: src/common/check-usage.js [hunk 3]
```diff
@@ -8,7 +8,9 @@ const checkUsage = function (manifest, varName) {
...
+      continue
+    }
     // Check if there is a key that corresponds to the variable name
     if (envVars['varName']) {
```
Comment (justmoon) [inline src/common/check-usage.js]: I stumbled over this when I was working with this library myself. I think this is a bug. You meant:

```js
if (envVars[varName]) {
```

File: test/generate-manifest.test.js [hunk 4]
```diff
@@ -162,6 +162,26 @@ describe('Generate Complete Manifest', function () {
     return expect(result).to.eventually.become(validManifest)
   })
 
+  it('should not throw an error the environment and vars fields are undefined', async function () {
```
Comment (justmoon) [inline test/generate-manifest.test.js]: Typo: +if

**Comment by justmoon** [comment]
Oops, just saw that you already merged #17. You should still rebase this to add in those tests!

**Comment by vtj2105** [comment]
Gonna close this PR. I went ahead and created a [separate PR](https://github.com/codius/manifest/pull/18) that is synced with the latest commits from @justmoon.

**Pull request closed without merging**